Tasks/DockerV1/containertag.ts

```diff
@@ -26,14 +29,14 @@ export function run(connection: ContainerConnection): Q.Promise<void> {
     } else {
         imageNames = [utils.getImageName()];
     }
-    
+    var qualifyImageName = tl.getBoolInput("qualifyImageName");
```

Comment:
Will it break existing tasks? What is the default value?

Reply:
The default value is `true`.
 
It could break the existing pipelines if the user has built the image with `qualifyImageName` set to `false` and hasn't set any value in the `tag` step. Because, earlier we weren't qualifying the image names, but we will be qualifying them now; by default.